### ISSUE TITLE
Conflict {maybe?} with Mount Journal Enhanced

### ISSUE THREAD

**Issue by nancikennedy** [Sonaza/DressUp#17]
Reported an issue to MJE's author, and did the testing he asked me to help him track down the problem. Even though the error report is showing MJE as the responsible party, it's having DressUp enabled that actually makes the error come up. See pasted response below...

[MJE ticket](https://www.curseforge.com/wow/addons/mount-journal-enhanced/issues/41): 

With just MJE and DressUp installed, got this error. In all combinations I tested, it was loading/unloading DressUp that changed whether an error was caught by BugSack. However, the error message has not fundamentally changed. I know that loading order can do wonky things to what is being reported as the problem, so it could be that your addon is unfairly getting the blame. Will cross-post to it's page.

 153x ...aceMountJournalEnhanced\UI\DressUpButton.lua:27: Usage: hooksecurefunc([table,] "function", hookfunc)
[C]: in function `hooksecurefunc'
...aceMountJournalEnhanced\UI\DressUpButton.lua:27: in function <...aceMountJournalEnhanced\UI\DressUpButton.lua:3>
...aceMountJournalEnhanced\UI\DressUpButton.lua:41: in function `ApplyPreviewButton'
...ddOns\MountJournalEnhanced\Settings\OptionsPanel.lua:111: in function <...ddOns\MountJournalEnhanced\Settings\OptionsPanel.lua:77>
[C]: ?
Masque\Libs\AceGUI-3.0\AceGUI-3.0-40.lua:72: in function <Masque\Libs\AceGUI-3.0\AceGUI-3.0.lua:70>
Masque\Libs\AceGUI-3.0\AceGUI-3.0-40.lua:290: in function `Fire'
...GUI-3.0\widgets\AceGUIContainer-BlizOptionsGroup.lua:32: in function <...GUI-3.0\widgets\AceGUIContainer-BlizOptionsGroup.lua:31>
[C]: in function `pcall'
FrameXML\InterfaceOptionsFrame.lua:206: in function <FrameXML\InterfaceOptionsFrame.lua:205>
[C]: in function `securecall'
FrameXML\InterfaceOptionsFrame.lua:229: in function <FrameXML\InterfaceOptionsFrame.lua:221>

**Comment by exochron** [comment]
Hey there. I'm going to fix this in MJE.
It's basically the same issue as in #16. I'm also trying to access DressUpFrame.ModelScene. That key was added with 8.2.5 into the Blizzard DressUp frames. It's really not nice to replace some frames entirelly ;)

**Comment by Sonaza** [comment]
Yes it's a little problematic to override so much but I didn't have the patience or the time to start learning how the model scene thing works or if it can even be used to implement all the features this addon has.

**Comment by exochron** [comment]
Maybe you should find the time then. Or you could find another solution which doesn't rely on overwriting DressUpFrame. Because there is a similar error, when you just preview a mount from a vendor or link. ;)

**Comment by Sonaza** [comment]
I'm not actively playing or even subscribed at the moment, and that's not even counting the fact addon developers aren't paid for the work so I don't think I _should_ really do anything.

Sorry it's causing problems but I at this moment don't care enough to fix anything in my addon.

Rather, it's Blizzard who should stop trampling on the feet of addon makers. Gotta keep churning out stupid changes after stupid changes without any official documentation.

**Comment by Meivyn** [comment]
This also happens when you try to sheathe/unsheathe weapons in the dressing room, when you preview a mount, and basically all things that are referring to ``ModelScene``.

It's sad you don't have the time or the motivation to fix this, your addon is helpful with transmog and I like to use it in combination with my Mog Assistant. 

I don't get either why you are overwriting it, instead of just creating frames on top of it.

I also hope Blizzard will make good documentation for add-ons developers in the future, just like they do with their web API with the Developer Portal.
